azure/Kqlmagic/draft_client.py

```diff
@@ -41,12 +41,27 @@ class DraftClient(object):
     _API_VERSION = "v1"
     _GET_SCHEMA_QUERY = ".show schema"
 
-    def __init__(self, conn_kv: dict, domain: str, data_source: str):
+    _CLOUD_AAD_URLS={
```

Comment:
what do you mean by "In case it is not the original public url, just leave it as is without modifying it"? in kusto client we dont support custom cloud urls